modules/upi-provisioning-new-hosts-in-a-upi-cluster.adoc

```diff
@@ -22,42 +22,182 @@ To provision bare-metal hosts to the cluster by using the BMO, you must set the
+<8> To skip certificate validation, set `disableCertificateVerification` to true.
+<9> Optional: Replace `<root_device_hint>` with a device path if you specify a root device hint.
+--
+
+* *DHCP configuration* `bmh.yaml`:
 +
 [source,yaml]
 ----
 ---
 apiVersion: v1
 kind: Secret
 metadata:
-  name: worker1-bmc
+  name: openshift-worker-<num>-bmc-secret <1>
   namespace: openshift-machine-api
 type: Opaque
 data:
-  username: <base64_of_uid>
-  password: <base64_of_pwd>
+  username: <base64_of_uid> <2>
+  password: <base64_of_pwd> <2>
 ---
 apiVersion: metal3.io/v1alpha1
 kind: BareMetalHost
 metadata:
-  name: worker1
+  name: openshift-worker-<num> <1>
   namespace: openshift-machine-api
 spec:
+  online: True
+  bootMACAddress: <nic1_mac_address> <3>
   bmc:
-    address: <protocol>://<bmc_url> <1>
-    credentialsName: "worker1-bmc"
-  bootMACAddress: <nic1_mac_address>
-  externallyProvisioned: false <2>
-  customDeploy:
-    method: install_coreos
-  online: true
-  userData:
-    name: worker-user-data-managed
-    namespace: openshift-machine-api
-----
-<1> You can only use bare-metal host drivers that support virtual media networking booting, for example `redfish-virtualmedia` and `idrac-virtualmedia`.
-<2> You must set the `spec.externallyProvisioned` specification in the `BareMetalHost` custom resource to `false`. The default value is `false`.
+    address: <protocol>://<bmc_url> <4>
+    credentialsName: openshift-worker-<num>-bmc-secret <1>
+    disableCertificateVerification: True <5>
+  rootDeviceHints:
+    deviceName: <root_device_hint> <6>
+----
++
+<1> Replace `<num>` for the machine and worker numbers of the bare metal nodes in the `name`, `credentialsName`, and the `preprovisioningNetworkDataName` fields.
++
+<2> Replace `<base64_of_uid>` and `<base64_of_pwd>` with the base64 string of the user name and password.
++
+<3> Replace `<nic1_mac_address>` with the MAC address of the bare metal node's first NIC. See the "BMC addressing" section for additional BMC configuration options.
++
+<4> Replace `<protocol>` with the BMC protocol, such as IPMI, RedFish, or others. Replace `<bmc_url>` with the URL of the bare metal node's baseboard management controller.
++
+<5> To skip certificate validation, set `disableCertificateVerification` to true.
++
+<6> Optional: Replace `<root_device_hint>` with a device path if you specify a root device hint.
++
+[NOTE]
+====
+If the MAC address of an existing bare metal node matches the MAC address of a bare metal host that you are attempting to provision, then the installation will fail. If the host enrollment, inspection, cleaning, or other steps fail, the Bare Metal Operator retries the installation continuously. See "Diagnosing a host duplicate MAC address" for more information.
+====
+
+. Create the bare metal node:
++
+[source,terminal]
+----
+$ oc -n openshift-machine-api create -f bmh.yaml
```

Comment:
nit: I don't think `oc create` requires a namespace

Reply:
What would be the correct command? `$ oc create -f bmh.yaml`?

Reply:
Correct

Reply:
Updated As Suggested.